docs/image-transfer-readme.md

```diff
@@ -0,0 +1,58 @@
+# Manual Workflow to Transfer Docker Images
+
+This workflow is designed to automate the process of transferring Docker images to a specified Docker Hub destination organization.
+It uses GitHub Actions to facilitate the image transfer.
+
+## Purpose
+The purpose of this workflow is to perform the following tasks:
+
+1. Update the configuration file config.yml with Docker Hub credentials and the destination organization. 
+2. Push Docker images to the destination organization on Docker Hub. 
+3. Notify the status of the workflow via Slack.
```

Comment:
are we notifying?
In case case yes.. use devops only for notifications to be triggered and only after failures.

Reply:
If it fails, then we will notify devops slack group